cmd/frontend/graphqlbackend/site.go

```diff
@@ -24,6 +24,8 @@ import (
 
 const singletonSiteGQLID = "site"
 
+var disableManagementConsoleAuth = env.Get("DISABLE_MANAGEMENT_CONSOLE_AUTH", "false", "When true, no password for accessing managment console is generated (and needed).")
```

Comment:
```suggestion
var disableManagementConsoleAuth = env.Get("DISABLE_MANAGEMENT_CONSOLE_AUTH", "false", "When true, no password for accessing managment console is generated nor needed.")
```

I'm not a native speaker, but... is it better?

Reply:
i took stephen's suggestion. they're both good, yours and his, thanks